### ISSUE TITLE
Increase Accessibility

### ISSUE THREAD

**Issue by Gazook89** [Gazook89/savage-bingo#13]
I suspect the site fails to meet Accessibility standards, since Accessibility wasn't really considered at all during initial development. 

1. Use a screen reader to check current Accessibility. 

2. Correct semantic html tag errors

3. Add aria labels where semantic tags won't work

4. Look at color contrasts and font sizes

Etc
